services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationController.kt

```diff
@@ -31,7 +32,9 @@ private val logger = KotlinLogging.logger {}
 @RequestMapping("/hakemukset")
 class ApplicationController(
```

Comment:
Should the other application endpoints have some access control as well?

Reply:
Added.